kuma/feeder/utils.py

```diff
@@ -176,7 +178,7 @@ def save_entry(feed, entry):
     if len(entry.guid) <= max_guid_length:
         entry_guid = entry.guid
     else:
-        entry_guid = md5(smart_str(entry.guid)).hexdigest()
+        entry_guid = md5(smart_str(entry.guid).encode('utf-8')).hexdigest()
```

Comment:
Would ``smart_bytes(entry.guid)`` work instead of ``smart_str(entry.guid).encode('utf-8')``?